queue_services/entity-filer/tests/unit/filing_processors/test_annual_report.py

```diff
@@ -54,14 +70,15 @@ def test_process_ar_filing(app, session):
     # TEST
     with freeze_time(now):
         filing = create_filing(payment_id, ar, business.id)
-        filing_id = filing.id
-        filing_msg = {'filing': {'id': filing_id}}
-        annual_report.process(business, filing.filing_json['filing'], filing_meta=filing_meta)
+        annual_report.process(business, filing.filing_json['filing'], filing_meta=filing_meta, flag_on=flag_on)
 
     # check it out
-    # NOTE: until we save or convert the dates, they are FakeDate objects, so casting to str()
-    assert str(business.last_agm_date) == str(agm_date)
-    assert str(business.last_ar_date) == str(agm_date)
+    if flag_on and in_dissolution and not eligibility:
+        assert batch_processing.status == BatchProcessing.BatchProcessingStatus.WITHDRAWN.value
+        assert batch_processing.notes == 'Moved back to good standing'
```

Comment:
we also need explicitly check there is no WITHDRAWN entry for the 'Not withdrawn from the dissolution process' scenario